lib/improver/spotdata/common_functions.py

```diff
@@ -347,6 +356,8 @@ def apply_bias(vertical_bias, dzs):
         available, otherwise it returns the whole set.
 
     """
+    dzs = np.array(dzs)
```

Comment:
Agreed. I've got rid of this and changed my input in the unit_test to be a numpy array as it should have been,